pkg/local-disk-manager/filter/filter_disk.go

```diff
@@ -119,6 +128,7 @@ func (ld *LocalDiskFilter) DevType() *LocalDiskFilter {
 
 func (ld *LocalDiskFilter) NoPartition() *LocalDiskFilter {
 	if len(ld.localDisk.Spec.PartitionInfo) > 0 {
+		ld.FailedMessage[v1alpha1.LocalDiskAssignFailReasonHasPartition] = struct{}{}
```

Comment:
when partition found in disk, the corresponding status should be Bound, so we may ignore this error too.